rubix-prestosql/src/main/java/com/qubole/rubix/prestosql/PrestoClusterManager.java

```diff
@@ -48,170 +28,51 @@
 /**
  * Created by stagra on 14/1/16.
  */
-public class PrestoClusterManager extends ClusterManager
+public class PrestoClusterManager extends SyncClusterManager
```

Comment:
We need the other form too because there is a standalone mode where Presto's NodeManager is not available. We should create a new one say SyncPrestoClusterManager as a copy of what you have and keep PrestoClusterManager based on AsyncClusterManager

Reply:
I created an implementation of Presto's NodeManager that can be used for standalone mode. We can move that out of the test side of things if you need it. This provides a much cleaner separation of concerns IMHO.